.aspect/bazelrc/bazel6.bazelrc

```diff
@@ -0,0 +1,15 @@
+# Speed up all builds by not checking if external repository files have been modified.
+# Docs: https://github.com/bazelbuild/bazel/blob/1af61b21df99edc2fc66939cdf14449c2661f873/src/main/java/com/google/devtools/build/lib/bazel/repository/RepositoryOptions.java#L244
+build --noexperimental_check_external_repository_files
+fetch --noexperimental_check_external_repository_files
+query --noexperimental_check_external_repository_files
```

Comment:
Could we simplify this to `common --noexperimental_check_external_repository_files`?

Reply:
@varungandhi-src those files are managed directly by Aspect, and they keep them updated. I don't know why they have to specify those three times, I think it's best you ask them on their support channel on our Slack.